cruise/CruiseInformation.csv

```diff
@@ -64,16 +80,19 @@ PS1610A,R/V Pacific Storm,2016-10-11T16:00:00,2016-10-12T02:00:00,Endurance CE02
 NBP16-09,R/V Nathaniel B. Palmer,2016-10-21T10:35:00,2016-11-15T19:00:00,Argentine Basin 3
 NBP16-10,R/V Nathaniel B. Palmer,2016-11-20T11:14:00,2016-12-16T09:55:00,Southern Ocean 3
 PS1612,R/V Pacific Storm,2016-12-15T00:00:00,2016-12-15T00:00:00,Endurance Gliders
+TS20161219,F/V Tenacious II,2016-12-19T00:00:00,2016-12-19T23:59:59,Pioneer glider cp_336 recovery
 PS1701A,R/V Pacific Storm,2017-01-05T00:15:00,2017-01-07T00:30:00,Endurance CE02SHSP and Gliders
 P7a,M/V Warren Jr.,2017-01-15T00:00:00,2017-01-19T23:59:59,Pioneer 7a #TO BE DELETED
 WJ20170115,M/V Warren Jr.,2017-01-15T00:00:00,2017-01-19T23:59:59,Pioneer 7a
 EK-1701,R/V Elakha,2017-01-31T18:30:00,2017-01-31T22:40:00,Endurance GL info
 PS1702A,R/V Pacific Storm,2017-02-13T17:29:00,2017-02-13T23:20:00,deployed CE02SHSP-00005 and Glider 381
+TS20170306,F/V Tenacious II,2017-03-06T00:00:00,2017-03-06T23:59:00,Pioneer glider cp_335 recovery
 DS17-03A,F/V Discovery,2017-03-08T07:05:00,2017-03-08T14:00:00,Glider recovery.
 PS1703C,R/V Pacific Storm,2017-03-31T00:00:00,2017-03-31T00:00:00,Glider recovery.
 SKQ201704S,R/V Sikuliaq,2017-04-09T19:20:00,2017-04-23T19:30:00,Endurance 7.  times added from 4\9 report and QuickLook document.
 P7b,M/V Scarlett Isabella,2017-04-18T00:00:00,2017-04-22T23:59:59,Pioneer 7b glider deployment/recovery #TO BE DELETED
 SI20170418,M/V Scarlett Isabella,2017-04-18T00:00:00,2017-04-22T23:59:59,Pioneer 7b glider deployment/recovery
+TS20170423,F/V Tenacious II,2017-04-23T00:00:00,2017-04-23T23:59:59,

```

Comment:
any note for what gliders were recovered and/or deployed?

Reply:
Agreed! Done